constants/index.ts

```diff
@@ -94,9 +94,10 @@ export const SYNC_TXS_JOBS_RETRY_DELAY = 2000
 export const BCH_TIMESTAMP_THRESHOLD = 1501588800 // 2017 Aug 1, 12PM
 export const XEC_TIMESTAMP_THRESHOLD = 1605398400 // 2020 Nov 15, 12AM
 
+export const DEFAULT_WORKER_LOCK_DURATION = 120000
 // Wait time (in ms) between sync of current prices
 export const CURRENT_PRICE_SYNC_DELAY = 60000
-export const DEFAULT_WORKER_LOCK_DURATION = 120000
+export const SUBSCRIBE_ADDRESSES_RETRY_DELAY = 60000
```

Comment:
better rename "retry" to "repeat", if I understood correctly, because "retry" conveys it failed and it will retry, whereas this is being used to determine the interval in which the job will repeat regardless of it's success.

Reply:
indeed, done